diff --git a/modules/paragraphs_i18n/paragraphs_i18n.info b/modules/paragraphs_i18n/paragraphs_i18n.info
index c2da534..e65e974 100644
--- a/modules/paragraphs_i18n/paragraphs_i18n.info
+++ b/modules/paragraphs_i18n/paragraphs_i18n.info
@@ -1,8 +1,5 @@
 name = Paragraphs - Internationalization
-description = Allow paragraph bundles to be duplicate for translation.
+description = Allow paragraphs items to be duplicated for translation.
 core = 7.x
 package = Paragraphs
-
-dependencies[] = paragraphs
-dependencies[] = replicate
 dependencies[] = replicate_paragraphs
diff --git a/modules/paragraphs_i18n/paragraphs_i18n.module b/modules/paragraphs_i18n/paragraphs_i18n.module
index c849853..7c340d3 100644
--- a/modules/paragraphs_i18n/paragraphs_i18n.module
+++ b/modules/paragraphs_i18n/paragraphs_i18n.module
@@ -1,26 +1,4 @@
 <?php
-
-/**
- * Implements hook_field_prepare_translation().
- *
- * @see field_attach_prepare_translation()
- */
-function paragraphs_i18n_field_prepare_translation($entity_type, $entity, $field, $instance, $langcode, &$items, $source_entity, $source_langcode) {
-    list($id, , ) = entity_extract_ids($entity_type, $entity);
-
-    if (empty($id)) {
-        $news_items = array();
-        foreach ($items as $key => &$item) {
-            if ($current_entity = paragraphs_field_get_entity($item)) {
-                $current_entity = replicate_clone_entity('paragraphs_item', $current_entity);
-                $current_entity->setHostEntity($entity_type, $source_entity, $langcode, FALSE);
-                $current_entity->save(TRUE);
-                $news_items[] = array(
-                    'value' => $current_entity->item_id,
-                    'revision_id' => $current_entity->revision_id,
-                );
-            }
-        }
-        $items = $news_items;
-    }
-}
\ No newline at end of file
+// This module's only purpose is to provide a dependency to
+// replicate_paragraphs. The actual implementation resides in
+// paragraphs_field_prepare_translation().
diff --git a/paragraphs.module b/paragraphs.module
index 0a01a5b..2671405 100644
--- a/paragraphs.module
+++ b/paragraphs.module
@@ -1216,18 +1216,16 @@ function paragraphs_field_attach_form($entity_type, $entity, &$form, &$form_stat
  * @see field_attach_prepare_translation()
  */
 function paragraphs_field_prepare_translation($entity_type, $entity, $field, $instance, $langcode, &$items, $source_entity, $source_langcode) {
-  if (!module_exists("paragraphs_i18n")) {
-      list($id, , ) = entity_extract_ids($entity_type, $entity);
-
-      // field_attach_prepare_translation() copied the entity ids from the source,
-      // as we need a new entity for a new translation, we cannot reuse that.
-      // @todo clone existing paragraphs to new translation
-      if (empty($id)) {
-          $items = array();
-      }
-  } else {
-      paragraphs_i18n_field_prepare_translation($entity_type, $entity, $field, $instance, $langcode, $items, $source_entity, $source_langcode);
+  if(module_exists('replicate_paragraphs')) {
+    replicate_paragraphs_clone_items($entity, $entity_type, $field['field_name'], $langcode);
+    $items = $entity->{$field['field_name']}[$langcode];
+    return;
   }
+
+  // field_attach_prepare_translation() copied the entity ids from the source,
+  // as we need a new entity for a new translation, we cannot reuse that.
+  // @todo clone existing paragraphs to new translation
+  $items = array();
 }
 
 /**
